cypress/integration/features/deals/create-open-proposal-wizard.feature

```diff
@@ -0,0 +1,46 @@
+Feature: Create Open Proposal
+  Background:
+    Given I navigate create open proposal wizard
+    
+  Scenario: Go back to deal type select
+    Given I am presented with Open Proposal proposal stage
+    When I go to previous step
+    Then I am presented the option to choose a partner
+
+  Scenario: I proceed from proposal stage without filling in required fields
+    When I try to proceed to next step
+    Then I am presented with Open Proposal proposal stage
+    And I am presented with errors proposal stage required errors
+
+  Scenario: I proceed from proposal stage after filling required fields with too short summary and description
+    When I fill in proposal title correctly
+    And I fill in proposal summary with text that is too short
+    And I fill in proposal description with text that is too short
```

Comment:
Re: reusable-step-definitions
https://github.com/TheBrainFamily/cypress-cucumber-preprocessor#reusable-step-definitions
```
    And I fill in proposal  with text that is too short
    And I fill in proposal description with text that is too short
```

In the test file you can reduce duplication

Note: Here I'm using regex matches, which is also supported 
https://github.com/cucumber/cucumber-expressions#custom-parameter-types (--> note the `regex` entry)
Here is a list of predefined "parameters": https://github.com/cucumber/cucumber-expressions#parameter-types

```diff
- When("I fill in proposal summary with text that is too short", () => {
+ When(/^I fill in (.*) with text that is too short$/, (fieldType) => {
  // Note: Because in description we use "proposal summary", but the selector is `proposal-summary-field`
  // We could either
  // 1. Define a separate map
  // 2. try to convert the description string to the selector string
```

Reply:
I can look into that